lib/active_interaction/filter.rb

```diff
@@ -205,12 +209,14 @@ def describe(value)
       "(Object doesn't support #inspect)"
     end
 
+    # @param interaction [Base, nil]
+    #
     # @return [Object]
-    def raw_default
+    def raw_default(interaction)
       value = options.fetch(:default)
 
       if value.is_a?(Proc)
-        value.call
+        interaction.instance_exec(&value) if interaction
```

Comment:
None of the tests fail if I remove it. It looks like only gets called with `nil` when it's not a proc:

```rb
filter.default(nil) if default && !default.is_a?(Proc)
```